src/game/common/const.mjs

```diff
@@ -122,6 +122,26 @@ export const GameTreasure = {
   rsg: { type: 'rsg', label: 'RSG', tile: 6, value: 200, baseVal: 1, denomination: 0 },
 };
 
+export const TEAMS = {
```

Comment:
let's not add more shit to this file - please create a separate file for the teams-related const